service/oncology/patient-oncology-summary-service.js

```diff
@@ -118,10 +115,10 @@ function getOncologyIntegratedProgramSnapshot(request) {
   let patientUuid = request.uuid;
   let queryParts = {
     columns:
-      "t1.encounter_id, DATE_FORMAT(t1.encounter_datetime,'%d%-%m%-%Y') as encounter_datetime, REPLACE(t3.name, 'Oncology ', '') AS `visit_name`, t5.name AS location, t6.breast_exam_findings, t6.via_test_result",
+      "t1.encounter_id, t1.encounter_datetime, t6.encounter_type_name, REPLACE(t3.name, 'Oncology ', '') AS `visit_name`, t5.name AS location, t6.breast_exam_findings, t6.via_test_result, t6.hiv_status, t6.prior_via_test_result, t6.prior_via_test_result_date",
```

Comment:
You've  `encounter_datetime` formatting. Is it already formatted or not need to format or to be done in the fronted side?

Reply:
I'm passing it through the DatePipe client-side.